packages/sanity/src/core/studio/components/navbar/navDrawer/LocaleMenu.tsx

```diff
@@ -4,9 +4,15 @@ import React, {useCallback} from 'react'
 import {useLocale} from '../../../../i18n/hooks/useLocale'
 import {Button} from '../../../../../ui-components'
 
+const LOCALE_SELECTION_DISABLED = true
```

Comment:
If this is a temporary change, a TODO comment may be helpful however I see that https://github.com/sanity-io/sanity/pull/5381 did not do that either lol